src/components/Search/SearchView/SearchView.css

```diff
@@ -0,0 +1,7 @@
+.gridStyle {
+  position: absolute ;
+  margin: '0 auto';
+  left: 0;
+  right: 0;
+  z-index: 1;
+}
```

Comment:
I'm unsure why this style was added, but it changes the rendering of the SearchInput. we can remove this and size the table using `<Grid>` instead